packages/catalog-model/src/location/helpers.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LocationSpec } from './types';
+
+/**
+ * Parses a string form location reference.
+ *
+ * @param ref A string-form location reference, e.g. 'url:https://host'
+ * @returns A location reference, e.g. { type: 'url', target: 'https://host' }
+ */
+export function parseLocationReference(ref: string): LocationSpec {
```

Comment:
Would say we should use a more specific type here that doesn't include `presence`, or do you think it's something we want?

Reply:
Hm yeah, I did that for the other function, probably makes sense to keep lean here too.

Reply:
At least until (and if!) we invent presence as part of the format